pkg/controller/constraintstatus/constraintstatus_controller.go

```diff
@@ -91,6 +92,16 @@ func PodStatusToConstraintMapper(packerMap handler.MapFunc) handler.MapFunc {
 			log.Error(fmt.Errorf("constraint status resource with no kind label: %s", obj.GetName()), "missing label while attempting to map a constraint status resource")
 			return nil
 		}
+		if selfOnly {
+			pod, ok := labels[v1beta1.PodLabel]
```

Comment:
A few questions:
* Are we already setting this label? If not, do we need to support a migration?
* When a pod terminates, does its status get cleaned up along with it, or will it be orphaned due to this new behavior?

Reply:
- We are already setting this label on the status resource as part of the original design: https://github.com/open-policy-agent/gatekeeper/blob/183f40aa6c06e3c5d04e58836b58631b7c9ab081/apis/status/v1beta1/constraintpodstatus_types.go#L100
- PodStatus objects have an owner reference to the gatekeeper pod, so K8s GC will delete any zombie status resources